TYPO3.Neos.Kickstarter/Resources/Private/Generator/TypoScript/NodeTypes/Pages/Page.ts2

```diff
@@ -0,0 +1,11 @@
+prototype(TYPO3.Neos.NodeTypes:Page.Document) < prototype(Neos:DefaultPage) {
```

Comment:
Hehe, who knows maybe it's _me_ who's wrong ;)
Anyways, I'd like to avoid dependencies where possible - but as you said, it's not an issue in this case probably!